### ISSUE TITLE
Copied text in the editor doesn't propogate from Linux VM to Windows

### ISSUE THREAD

**Issue by Molochnik** [elfmz/far2l#1985]
Prerequisites: Linux virtual guest machine (Linux Mint specifically), Windows - host.
When you copy a text in the editor and want to paste it in the same Linux machine everything works fine.
But if you try to paste it in the host Windows machine the copy buffer contains an old text (previously copied).
All other text editors and viewers including web browsers, text editors and terminals work with text copy&paste between
Linux<->Windows properly.

**Comment by akruphi** [comment]
If you use Linux in WSL or with Wayland you may try run far2l in xWayland mode:
running from console: `GDK_BACKEND=x11 far2l`
(see details in far2l help by F1 in section `FAR2L features - Getting Started`)

If it not solve problem please inform details about your VM (WSL, VirtualBox, ...)

**Comment by Molochnik** [comment]
No I use linux mint 20.3 (not the latest) virtual machine on VMWare player

**Comment by unxed** [comment]
May be this is somehow related?
https://github.com/wxWidgets/wxWidgets/issues/24391#issuecomment-1989195956

Also maybe separate issue to VMWare player bug tracker is needed.

**Comment by unxed** [comment]
Check now, please!

**Comment by Molochnik** [comment]
Copy/paste works perfect, thank you!

**Comment by Molochnik** [comment]
Sorry was too quick, no doesn't work, just checked copy/paste from the Linux text editor - ts works fine. text is copied to the shared VMWare buffer, far2l copies only to local buffer.

**Comment by Molochnik** [comment]
But sometimes it works! I don't understand what does it depend on

**Comment by unxed** [comment]
Check now on recent master please!

**Comment by Molochnik** [comment]
Compilation was successful but the linking is failed:
```

l but the CMakeFiles/NetRocks-FTP.dir/src/Protocol/FTP/FTPConnection.cpp.o: In function `OpenSSLContext::sNewClientSessionCB(ssl_st*, ssl_session_st*)':
FTPConnection.cpp:(.text._ZN14OpenSSLContext19sNewClientSessionCBEP6ssl_stP14ssl_session_st+0x34): undefined reference to `SSL_SESSION_up_ref'
CMakeFiles/NetRocks-FTP.dir/src/Protocol/FTP/FTPConnection.cpp.o: In function `OpenSSLContext::OpenSSLContext(StringConfig const&)':
FTPConnection.cpp:(.text._ZN14OpenSSLContextC2ERK12StringConfig+0x40): undefined reference to `TLS_client_method'

```

**Comment by unxed** [comment]
Can you please share information about ditro you are building on?

**Comment by Molochnik** [comment]
I tried to build it on Mint 20.3 and CentOS8 stream

**Comment by unxed** [comment]
Do not reproduce. Try building from scratch, in a clean folder.

**Comment by Molochnik** [comment]
there was no internet and upgrade was impossible After inet appeared I updated all linuxes ant everything went fine. Compilation and linking are successful copy clipboard works as well perfectly, thanks! Wveytings id Ok now.
BTW how to see the current version build? --version doesnt show it

**Comment by unxed** [comment]
F1 or window title

**Comment by unxed** [comment]
Can this issue be closed now?

**Comment by Molochnik** [comment]
Yes!
